### DISCUSSION TITLE
Prominent relationship

### DISCUSSION THREAD

**Discussion by zheng-xd** [dineug/erd-editor#276] [Ideas]
Your work is very good, I compared dozens of tools, your work is the most suitable, very friendly to developers and readers.

I have tried trevor's ERD function before. The most attractive point for me is to highlight the relationship. This function is very useful when there are a lot of tables. It is very intuitive. I hope you can use it.

thanks

https://trevor.io/create-an-er-diagram-online/
![image](https://user-images.githubusercontent.com/76488150/132829855-cf6266a3-24eb-4296-b11f-c014516b4a5d.png)

**Comment by dineug** [comment]
#446
